src/types.ts

```diff
@@ -0,0 +1,6 @@
+
+export interface Config {
+
+    host: string;
```

Comment:
This project is configured to use indentation of 2 spaces not 4. Can you update these so they are indented with two spaces?

Reply:
okay! done